subiquity/common/errorreport.py

```diff
@@ -140,6 +141,9 @@ def _bg_add_info():
                 self.pr.add_hooks_info(None)
                 apport.hookutils.attach_hardware(self.pr)
             self.pr['Syslog'] = apport.hookutils.recent_syslog(re.compile('.'))
+            snap_name = os.environ.get('SNAP_NAME', None)
+            if bool(snap_name):
```

Comment:
did you mean?
```python
if snap_name is not None:
```

`if bool(snap_name)` is equivalent to `if snap_name`.

Reply:
I was intentionally excluding the empty string as well.